cluster-operator/src/main/java/io/strimzi/operator/cluster/model/cruisecontrol/Capacity.java

```diff
@@ -103,72 +194,32 @@ private JsonObject generateJbodDiskCapacity(Storage storage, int idx) {
         if (storage instanceof PersistentClaimStorage) {
             return getSizeInMiB(((PersistentClaimStorage) storage).getSize());
         } else if (storage instanceof EphemeralStorage) {
             if (((EphemeralStorage) storage).getSizeLimit() != null) {
                 return getSizeInMiB(((EphemeralStorage) storage).getSizeLimit());
             } else {
-                return DEFAULT_BROKER_DISK_CAPACITY_IN_MIB;
-            }
-        } else if (storage instanceof JbodStorage) {
-            // The value generated here for JBOD storage is used for tracking the total
-            // disk capacity per broker. This will NOT be used for the final disk capacity
-            // configuration since JBOD storage requires a special disk configuration.
-            List<SingleVolumeStorage> volumeList = ((JbodStorage) storage).getVolumes();
-            double size = 0;
-            for (SingleVolumeStorage volume : volumeList) {
-                size += generateDiskCapacity(volume);
+                return Broker.DEFAULT_DISK_CAPACITY_IN_MIB;
             }
-            return size;
         } else {
             throw new IllegalStateException("The declared storage '" + storage.getType() + "' is not supported");
         }
     }
 
-    /**
-     * Generate a capacity configuration for cluster
-     *
-     * @return Cruise Control capacity configuration as a String
-     */
-    public String generateCapacityConfig() {
-        JsonArray brokerList = new JsonArray();
-        if (storage instanceof JbodStorage) {
-            // A capacity configuration for a cluster with a JBOD configuration
-            // requires a distinct broker capacity entry for every broker because the
-            // Kafka volume paths are not homogeneous across brokers and include
-            // the broker pod index in their names.
-            for (int idx = 0; idx < replicas; idx++) {
-                JsonObject diskConfig = new JsonObject().put("DISK", generateJbodDiskCapacity(storage, idx));
-                JsonObject brokerEntry = generateBrokerCapacity(idx, diskConfig, "Capacity for Broker " + idx);
-                brokerList.add(brokerEntry);
-            }
-        } else {
-            // A capacity configuration for a cluster without a JBOD configuration
-            // can rely on a generic broker entry for all brokers
-            JsonObject diskConfig = new JsonObject().put("DISK", String.valueOf(diskMiB));
-            JsonObject defaultBrokerCapacity = generateBrokerCapacity(DEFAULT_BROKER_ID, diskConfig, DEFAULT_BROKER_DOC);
-            brokerList.add(defaultBrokerCapacity);
-        }
-        JsonObject config = new JsonObject();
-        config.put("brokerCapacities", brokerList);
-
-        return config.encodePrettily();
-    }
-
     /*
      * Parse a K8S-style representation of a disk size, such as {@code 100Gi},
      * into the equivalent number of mebibytes represented as a Double.
```

Comment:
Should it be `represented as a String` now?
Same below

Reply:
I think so, if we do the String conversion here then we don't have to do it in the calling class

Reply:
So let's update the comments for the methods

Reply:
This one still says `represented as a Double.` even though we changed the return type to `String`